how-we-do-hiring-and-hr-at-posthog.md

```diff
@@ -0,0 +1,112 @@
+
+We get a lot of under-qualified candidates for many our roles. This is mainly due to the nature of remote working - when you open up access to everyone, the numbers of good (and poor) candidates both go up. We also have to be careful about our phrasing depending on which location we’ve placed a job ad in. For example, a marketing executive in the UK is essentially an intern fresh out of school, whereas a marketing executive in the US belongs in the C-suite. Ultimately, if they’re not a ‘strong yes,’ they’re a ‘no.’ And if we keep getting poor-fit candidates from a particular platform, we cease all ads on that platform.
+
+For those who look good on paper, the next step is an informal 30-minute Zoom call with Eltje.
+
+The first thing we’re looking for in a candidate is energy: a function of passion, enthusiasm, and a willingness to ask all the right questions in an interview. Sure, not everyone is extroverted, but curious intelligence can substitute for high energy where needed. Eltje makes notes about this stuff in initial interviews - in addition to notes about a candidate’s motivation to work _specifically_ for PostHog. Many candidates write generic cover letters when applying to different companies, but a better way to stand out is to reference a section of [our handbook](https://posthog.com/handbook) or a pull request we recently published on our [GitHub repo](https://github.com/PostHog/posthog/pulls).
+
+#### What we’re looking for
+
+We want candidates with transferable skills and a core set of personality traits: communicative, curious, fast learners who are able to work well with other people. You can teach new skills, but altering someone’s personality is a lot harder. Pick someone who has worked in a similar role, company, or product before, if only for the prior exposure that’ll prove useful to their new role at your company. 
+
+We’re also big on autonomy. A candidate who isn’t comfortable with autonomy over their role and output is a poor fit for us because we don’t believe in [micromanagement](https://posthog.com/handbook/company/management), death by [meetings](https://posthog.com/blog/meetings), or asking for permission to do what’s best for the company. We also [pivot](https://posthog.com/blog/story-about-pivots) a lot, choosing to run lots of mini-experiments to see what works. New hires thus need to be comfortable with the fast pace of change at PostHog.
+
+#### Next steps
+
+After the intro interview, the next step is a technical interview based on the role. New engineering hires might undergo a 90-minute, 2-part interview with Tim and one of the other engineers. New design hires might chat to Cory and undergo a portfolio review, while new marketing hires might chat to James, Mo, and Joe.
```

Comment:
This sounds very formal, I'd phrase it a bit more informally

Reply:
Would appreciate phrasing suggestions!